addons/html_editor/static/src/main/link/link_selection_odoo_plugin.js

```diff
@@ -1,15 +1,14 @@
 import { Plugin } from "@html_editor/plugin";
 import { isBlock } from "@html_editor/utils/blocks";
+import { selectElements } from "@html_editor/utils/dom_traversal";
 
 export class OdooLinkSelectionPlugin extends Plugin {
     static id = "odooLinkSelection";
     /** @type {import("plugins").EditorResources} */
     resources = {
         ineligible_link_for_zwnbsp_predicates: [
             (link) =>
-                [link, ...link.querySelectorAll("*")].some(
-                    (el) => el.nodeName === "IMG" || isBlock(el)
-                ),
+                [...selectElements(link, "*")].some((el) => el.nodeName === "IMG" || isBlock(el)),
```

Comment:
Why the `[...selectElements(...)]` then?

Reply:
What do you mean?